c2corg_api/views/image.py

```diff
@@ -2,11 +2,12 @@
 
 from c2corg_api.models.image import Image, schema_image, schema_update_image
 from c2corg_api.views.document import DocumentRest
-from c2corg_api.views import json_view
+from c2corg_api.views import json_view, cors_policy
 from c2corg_api.views.validation import validate_id
 
 
-@resource(collection_path='/images', path='/images/{id}')
+@resource(collection_path='/images', path='/images/{id}',
```

Comment:
We could use annotations like: `@public_json_view` and `@restricted_json_view` instead of `@json_view`.